tendrl.yml

```diff
@@ -1,69 +0,0 @@
----
-
-#
-# Install Tendrl server
-#
-
-- hosts: usm_server
-  remote_user: root
-  vars:
-    etcd_fqdn: "{{ ansible_fqdn }}"
-    etcd_ip_address: "{{ ansible_default_ipv4.address }}"
-    graphite_fqdn: "{{ etcd_fqdn }}"
-    etcd_tls_client_auth: True
-    etcd_trusted_ca_file: "/etc/pki/tls/certs/ca-usmqe.crt"
-  roles:
-    - role: qe-ssl-cert
-      ssl_cert_name: "etcd"
-      ssl_key_perm: "0644"
-      when: etcd_tls_client_auth|bool == True
-    - { role: epel, epel_enabled: 1, when: ansible_distribution == 'CentOS' }
-    - role: tendrl-ansible.grafana-repo
-      when: ansible_distribution == 'CentOS'
-    - role: qe-tendrl-repo
-      when: ansible_distribution == 'CentOS'
-    #- tendrl-ansible.ceph-installer # TODO: make it optional (when we don't deploy ceph)
-    - tendrl-ansible.tendrl-server
```

Comment:
The new playbook contains just `qe-ssl-cert` role, where did the other roles (such as `qe-tendrl-repo`) go?

Reply:
@mbukatov `qe-tendrl-repo` role is not required here anymore, because we are using repos added by *tendrl-ansible* `site.yml`.